api/tests/conftest.py

```diff
@@ -12,6 +15,8 @@
 DEFAULT_JOB_ID = 'myJobId'
 DEFAULT_USERNAME = 'test_username'
 
+CMR_URL_RE = re.compile('https://cmr.earthdata.nasa.gov/search/granules.json.*')
```

Comment:
might put `CMR_URL = 'https://cmr.earthdata.nasa.gov/search/granules.json'` in `__init__.py` then import it in both `handlers.py` and `conftest.py`